docs/modules/ROOT/pages/how-tos/exoscale/decommission.adoc

```diff
@@ -0,0 +1,95 @@
+= Uninstallation on Exoscale
+
+[abstract]
+--
+Steps to remove an OpenShift 4 cluster from https://exoscale.com[Exoscale].
+--
+
+[NOTE]
+--
+- The commands are idempotent and can be retried if any of the steps fail.
+- In the future, this procedure will be mostly automated
+--
+
+== Prerequisites
+
+* Exoscale https://community.exoscale.com/documentation/iam/quick-start/#api-keys[API key]
+* `terraform` >= 0.14
+* `exo` >= v1.14.0, https://community.exoscale.com/documentation/tools/exoscale-command-line-interface[Exoscale CLI]
+
+== Cluster Decommission
+
+. Export the following vars
++
+[source,console]
+----
+export EXOSCALE_ACCOUNT=<exoscale-account>
+export EXOSCALE_API_KEY=<exoscale-key>
+export EXOSCALE_API_SECRET=<exoscale-secret>
+export CLUSTER_ID=<cluster-name>
+export BASE_DOMAIN=<cluster-base-domain>
+export EXOSCALE_REGION=<cluster-region>
+
+export SSH_PUBLIC_KEY=~/.ssh/id_ed25519.pub
+
+export AWS_ACCESS_KEY_ID=${EXOSCALE_API_KEY}
+export AWS_SECRET_ACCESS_KEY=${EXOSCALE_API_SECRET}
+export AWS_S3_ENDPOINT="sos-${EXOSCALE_REGION}.exo.io"
+export TF_VAR_lb_exoscale_api_key=${EXOSCALE_API_KEY}
+export TF_VAR_lb_exoscale_api_secret=${EXOSCALE_API_SECRET}
+export TF_VAR_bootstrap_bucket="https://sos-${EXOSCALE_REGION}.exo.io/${CLUSTER_ID}-bootstrap/"
+export TF_VAR_ignition_ca=''
+export TF_VAR_rhcos_template="rhcos-4.7.7"
+export TF_VAR_cluster_id=$CLUSTER_ID
+export TF_VAR_base_domain=${BASE_DOMAIN}
+export TF_VAR_ssh_key="$(cut -d' ' -f1,2 <$SSH_PUBLIC_KEY)"
+----
+
+. Delete resources from clouscale.ch using Terraform
++
+[source,console]
+----
+cat > backend.tf <<EOF
+terraform {
+  backend "s3" {
+    key                         = "cluster.tfstate"
+    region                      = "us-east-1" # Ignored
+    bucket                      = "${CLUSTER_ID}-tf-state"
+    skip_credentials_validation = true
+    skip_metadata_api_check     = true
+  }
+}
+EOF
+terraform init
+terraform destroy
+----
+
+. Use Exoscale CLI tool to remove buckets
++
+[source,console]
+----
+mkdir -p ~/.config/exoscale
+cat <<EOF >> ~/.config/exoscale/exoscale.toml
+
+[[accounts]]
+  account = "${EXOSCALE_ACCOUNT}"
+  defaultZone = "${EXOSCALE_REGION}"
+  endpoint = "https://api.exoscale.ch/v1"
+  name = "${CLUSTER_ID}"
+EOF
+
+exo sos delete -r "${CLUSTER_ID}-bootstrap"
+exo sos delete -r "${CLUSTER_ID}-tf-state"
+----
+
+[NOTE]
+--
+This how-to currently doesn't contain instructions to decommission the cluster in Project Syn.
+If you registered the cluster in Project Syn, the places to look for data to decommission are:
+* Vault
+* Lieutenant
+* Syn Tenant Repo
+* VSHN LDAP
+* LDAP IP allowlist
+* Any DNS records created in VSHN DNS
```

Comment:
Not sure if we should document VSHN specifics here or if that should go into https://kb.vshn.ch/vshnsyn/ ?

Reply:
The same kinds of things are documented in the Cloudscale decommissioning instructions, cf. https://kb.vshn.ch/oc4/how-tos/cloudscale/decommission.html

Reply:
Ok. fine for me to leave it for now